lib/output.js

```diff
@@ -67,7 +67,8 @@ function OutputStream(options) {
         screw_ie8        : false,
         preamble         : null,
         quote_style      : 0,
-        keep_quoted_props: false
+        keep_quoted_props: false,
+        ecma5            : true
```

Comment:
The `bin/uglifyjs` command line flag should be `--ecma5`, but this property should be `ecma` with default value `5`.